src/lib/translate-ipfs-redirect-url.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * If you host helia-service-worker-gateway on an IPFS domain, the redirects file will route some requests from
+ * `<domain>/<wildcard-splat>` to `https://<domain>/?helia-sw=<wildcard-splat>`.
```

Comment:
```suggestion
 * If you host helia-service-worker-gateway on an IPFS domain, the _redirects file will route some requests from
 * `<domain>/<wildcard-splat>` to `https://<domain>/?helia-sw=<wildcard-splat>` if they hit the server instead of the service-worker.
```